Exception when calling `clusternator describe-services`

```sh
clusternator describe-services -vvv                                                                    Tue  8 Mar 2016 17:55:26 EST
debug: Winston Error Cannot find module '/Users/sdavenpo/.npm/lib/node_modules/clusternator/.private/config.local.json'
verbose: No "local" config found in /Users/sdavenpo/.npm/lib/node_modules/clusternator/.private/config.local.json, checking project
debug: Winston Error Cannot find module '/Users/sdavenpo/.npm/lib/node_modules/clusternator/.privateconfig.json'
verbose: No "project" config found in /Users/sdavenpo/.npm/lib/node_modules/clusternator/.privateconfig.json, checking global
debug: Winston Error Cannot find module '/etc/clusternator/config.json'
verbose: No "global" config found in /etc/clusternator/config.json
debug: Winston Error Cannot find module '/Users/sdavenpo/.npm/lib/node_modules/clusternator/.private/credentials.local.json'
verbose: No "local" credentials found in /Users/sdavenpo/.npm/lib/node_modules/clusternator/.private/credentials.local.json, checking project
debug: Winston Error Cannot find module '/Users/sdavenpo/.npm/lib/node_modules/clusternator/.privatecredentials.json'
verbose: No "project" credentials found in /Users/sdavenpo/.npm/lib/node_modules/clusternator/.privatecredentials.json, checking global
debug: Winston Error Cannot find module '/etc/clusternator/credentials.json'
verbose: No "global" credentials found in /etc/clusternator/credentials.json, checking environment variables
Error: Cannot read property 'then' of undefined TypeError: Cannot read property 'then' of undefined
    at /Users/sdavenpo/.npm/lib/node_modules/clusternator/lib/api/0.1/cli/cloud-service.js:24:49
    at _fulfilled (/Users/sdavenpo/.npm/lib/node_modules/clusternator/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/sdavenpo/.npm/lib/node_modules/clusternator/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/sdavenpo/.npm/lib/node_modules/clusternator/node_modules/q/q.js:796:13)
    at /Users/sdavenpo/.npm/lib/node_modules/clusternator/node_modules/q/q.js:604:44
    at runSingle (/Users/sdavenpo/.npm/lib/node_modules/clusternator/node_modules/q/q.js:137:13)
    at flush (/Users/sdavenpo/.npm/lib/node_modules/clusternator/node_modules/q/q.js:125:13)
    at nextTickCallbackWith0Args (node.js:452:9)
    at process._tickCallback (node.js:381:13)
```

--- a/src/api/0.1/cli/cli-api.js
+++ b/src/api/0.1/cli/cli-api.js
@@ -68,7 +68,6 @@ module.exports = {
     serve,
     listAuthorities,
     listProjects,
-    describeServices,
     build,
     deploy,
     stop,
@@ -301,15 +300,6 @@ function listProjects(yargs) {
     (y) => aws.listProjects()
       .then((projectNames) => projectNames.forEach(console.log))
       .fail(util.cliErr('Error listing projects.'))
-      .done());
-}
-
-function describeServices(yargs) {
-  return yargs.command(
-    'describe-services',
-    'Describe project services',
-    () => aws.describeServices()
-      .fail(util.cliErr(`Error describing services`))
       .done());
 }
 

